pkg/informer/informer.go

```diff
@@ -44,6 +75,162 @@ func InformerForResource(res schema.GroupVersionResource) (GenericInformer, erro
 	return i, nil
 }
 
+func isResourceAllowed(cm *v1.ConfigMap, group, kind string, allowedList []Resource, deniedList []Resource) bool {
+
+	var boolVar bool
+
+	// Ignore clusters and clusterstatus resources because these are handled by the aggregator.
+	// Ignore oauthaccesstoken resources because those cause too much noise on OpenShift clusters.
+	// Ignore projects as namespaces are overwritten to be projects on Openshift clusters - they tend to share
+	// the same uid.
+	list := []string{"events", "projects", "clusters", "clusterstatuses", "oauthaccesstokens"}
+
+	//remove all apiResources with kind in list
+	for _, name := range list {
+		if kind == name {
+			// klog.Info("list of names, resource kind:", name, kind)
+			return false
+		}
+	}
+
+	// remove denied resources from all apigroups if * otherwise remove from specific apigroups.
+	for _, deny := range deniedList {
+		for _, api := range deny.ApiGroups {
+			for _, rec := range deny.Resources {
+				if api == "*" && rec != "*" {
+					if rec == kind {
+						klog.Info("deny all group, specific resource", group, api, kind, rec)
```

Comment:
Let's change the verbosity of the logs in this function `klog.V(1).Info(...)`

Message can be more descriptive.
```suggestion
						klog.V(1).Info("Excluding resource because it matches deny list. ", group, api, kind, rec)
```

Reply:
Or 
```
klog.V(1).Infof("Resource %s %s matched deny list. ", group, kind)
```